application/org.openjdk.jmc.flightrecorder.flameview/src/main/js/flameviewColoring.js

```diff
@@ -130,7 +133,15 @@ const evaluateSearchElement = function(text) {
 };
 
 const removeSpecialCharacters = function(text) {
-	return text.trim().replace(/[&\/\\#,+()$~%'":*?<>{}]/g, '_').toLowerCase();
+	//return text.trim().replace(/[&\/\\#,+()$~%'":*?<>{}]/g, '_').toLowerCase();
+	const map = Array.prototype.map;
+    const resultText = map.call(text.trim().toLowerCase(), element => {
```

Comment:
indent level is off, and the following code block has some spacing to fix up

Reply:
;) it's not there